lib/spack/llnl/util/filesystem.py

```diff
@@ -38,6 +39,7 @@
 import getpass
 from contextlib import contextmanager, closing
 from tempfile import NamedTemporaryFile
+from subprocess import check_output as run
```

Comment:
Travis is complaining about this, I think.  Can you look at the broken travis build?